open-metadata-implementation/access-services/data-manager/data-manager-api/src/main/java/org/odpi/openmetadata/accessservices/datamanager/properties/APIManagerProperties.java

```diff
@@ -18,8 +18,6 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class APIManagerProperties extends SoftwareCapabilitiesProperties
 {
-    private static final long serialVersionUID = 1L;
```

Comment:
I wonder why you are removing this. [https://howtodoinjava.com/java/serialization/serialversionuid/](https://howtodoinjava.com/java/serialization/serialversionuid/ ) indicates that specifying is good practice.

Reply:
Because I am removing the face that these beans are serializable